include/oneapi/dpl/experimental/kt/esimd_radix_sort_cooperative.h

```diff
@@ -365,9 +365,16 @@ void cooperative(sycl::queue __q, _Range&& __rng, ::std::size_t __n) {
     }
     assert(__groups <= MAX_GROUPS);
 
-    auto p_sync = sycl::malloc_device<::std::uint32_t>(1024 + (__groups+2) * BIN_COUNT, __q);
+    const size_t full_buffer_size_sync = sizeof(::std::uint32_t) * (1024 + (__groups + 2) * BIN_COUNT);
+    const size_t full_buffer_size_tmp = sizeof(KeyT) * (__groups * __group_block_size);
+    const size_t full_buffer_size = full_buffer_size_sync + full_buffer_size_tmp;
```

Comment:
Would you mind replacing the variable names?
 - full_buffer_size_sync -> sync_buffer_size
 - full_buffer_size_tmp -> output_buffer_size
 - full_buffer_size -> temp_buffer_size